R/summarise_country.R

```diff
@@ -0,0 +1,163 @@
+summarise_sheet_indicator_list <- function(billion) {
+  billion <- rlang::arg_match(billion)
+  indicator_order_b <- indicator_order %>%
+    dplyr::filter(ind %in% billion_ind_codes(billion)) %>%
+    dplyr::arrange(order) %>%
```

Comment:
What is order?

Reply:
All right, as you indicate in a comment below, this data frame should already be ordered as it's a package export, so no need for that.